src/util/__tests__/luxon-utils.test.const.ts

```
@@ -0,0 +1,55 @@
+        input: ["2025-03-10/2025-03-12", "2025-03-11/2025-03-13", "2026-03-10/2026-03-12", "2026-03-11/2026-03-13"].map(
+            toInterval,
+        ),
+        output: ["2025-03-10/2025-03-13", "2026-03-10/2026-03-13"].map(toInterval),
+    },
+];
+
+/**
+ * Converts an ISO 8601 string into a validated Luxon Interval.
+ * @param iso - The ISO 8601 string representing the interval.
+ * @returns The valid Luxon Interval derived from the provided string.
+ * @throws {Error} If the ISO string does not produce a valid interval.
```

ESLint

[error] tsdoc-malformed-inline-tag: Expecting a TSDoc tag starting with "{@"

(tsdoc/syntax)